k8s/k8s.go

```diff
@@ -1788,6 +1829,30 @@ func (k *k8sOps) GetPersistentVolumeClaim(pvcName string, namespace string) (*v1
 		Get(pvcName, meta_v1.GetOptions{})
 }
 
+func (k *k8sOps) GetPersistentVolumeClaims(namespace string) (*v1.PersistentVolumeClaimList, error) {
+	if err := k.initK8sClient(); err != nil {
+		return nil, err
+	}
+
+	return k.client.Core().PersistentVolumeClaims(namespace).List(meta_v1.ListOptions{})
+}
+
+func (k *k8sOps) GetPersistentVolume(pvName string) (*v1.PersistentVolume, error) {
```

Comment:
question: aren't PVs also namespaced? if yes, why don't the k8s api need the namespace to get or list the PVs?

Reply:
PVs aren't namespaced. So they are not needed. Removed namespace for PV functions.